client/dashboard/app/style.scss

```diff
@@ -46,6 +47,25 @@
 	}
 }
 
+// Remove the top padding of the card body when the DataViews is empty.
+.components-card-body:has( > .dataviews-wrapper > .dataviews-no-results ) {
```

Comment:
How about creating a new file to do the override 🤔

Reply:
Hmmm any suggestions?

Reply:
Introduced dataforms-overrides.sccs and dataviews-overrides.scss in 7e71f699ecd377cd4d60900e3fa87a1da37e7f16